### PR TITLE
Add password update page

### PR DESCRIPTION
**Pull request by fgren**

### CODE REVIEW

File: login/login-update-password.ftl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                                data-icon-show="${properties.kcFormPasswordVisibilityIconShow!}" data-icon-hide="${properties.kcFormPasswordVisibilityIconHide!}"
+                                data-label-show="${msg('showPassword')}" data-label-hide="${msg('hidePassword')}">
+                            <i class="${properties.kcFormPasswordVisibilityIconShow!}" aria-hidden="true"></i>
+                        </button>
+                    </div>
+
+                    <#if messagesPerField.existsError('password')>
+                        <span id="input-error-password" class="${properties.kcInputErrorMessageClass!}" aria-live="polite">
+                            ${kcSanitize(messagesPerField.get('password'))?no_esc}
+                        </span> <!--FIX THIS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! -->
```
Comment (Tomas-Ka) [inline login/login-update-password.ftl]: Is this something worth dealing with before merging? Or would that be too much work

**Review by Tomas-Ka — CHANGES_REQUESTED** [review]
Looks good, just one slightly concerning comment
In the live environment the page shows white text on a white background when typing in a password. Is this fixed?

File: login/login-update-password.ftl (hunk 1 above)
Comment (fgren) [reply login/login-update-password.ftl]: hehe oops

**Review by Tomas-Ka — APPROVED** [review]
looks good!

**Pull request merged**